static-instrumenter/agent-instrumenter/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 plugins {
   id("otel.java-conventions")
   id("com.github.johnrengelman.shadow") version "7.1.2"
-  id("org.unbroken-dome.test-sets") version "4.0.0"
+  id("jvm-test-suite")
```

Comment:
I don't think this is needed, at least we don't have it in instrumentation repo (must be automatically part of gradle?)
```suggestion
```